protocols/kad/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # 0.40.0 [unreleased]
 
+- Add support for multiple protocol names. Update `Kademlia`, `KademliaConfig`,
+  and `KademliaProtocolConfig` accordingly.
+  See [issue 2837](https://github.com/libp2p/rust-libp2p/issues/2837).
```

Comment:
Can you reference the pull request as well?

Also for the sake of consistency, please follow the link convention like e.g.:

```
- Limit # of inbound streams to 32. [See PR 2699].

[PR 2699]: https://github.com/libp2p/rust-libp2p/pull/2699
```

Reply:
Fixed.